# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmInstance.h
# Commit: 25adf505426d
# Full Hash: 25adf505426d21f67e15d146f879b0ea53d343a7
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-23 04:09:13
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r 43f7556db2e1 -r 25adf505426d js/src/wasm/WasmInstance.h
--- a/js/src/wasm/WasmInstance.h	Tue Mar 22 19:44:13 2022 +0000
+++ b/js/src/wasm/WasmInstance.h	Tue Mar 22 19:59:05 2022 +0000
@@ -431,13 +431,12 @@
                            uint32_t dstTableIndex, uint32_t srcTableIndex);
   static int32_t tableFill(Instance* instance, uint32_t start, void* value,
                            uint32_t len, uint32_t tableIndex);
-  static void* tableGet(Instance* instance, uint32_t index,
-                        uint32_t tableIndex);
+  static void* tableGetFunc(Instance* instance, uint32_t index,
+                            uint32_t tableIndex);
   static uint32_t tableGrow(Instance* instance, void* initValue, uint32_t delta,
                             uint32_t tableIndex);
-  static int32_t tableSet(Instance* instance, uint32_t index, void* value,
-                          uint32_t tableIndex);
-  static uint32_t tableSize(Instance* instance, uint32_t tableIndex);
+  static int32_t tableSetFunc(Instance* instance, uint32_t index, void* value,
+                              uint32_t tableIndex);
   static int32_t tableInit(Instance* instance, uint32_t dstOffset,
                            uint32_t srcOffset, uint32_t len, uint32_t segIndex,
                            uint32_t tableIndex);